instructions-for-devs.md

```diff
@@ -0,0 +1,203 @@
+# Instructions for Developers & Testers
+
+This document contains `xud` setup instructions for developers and everyone with a need for more fine-grained control when running `xud`.
+
+## 1. Install
+
+First, clone the repository from GitHub and install dependencies. 
+
+Developers & Testers:
+```bash
+git clone https://github.com/ExchangeUnion/xud
+cd xud
+npm install
+```
+For everyone else, we recommend using the [latest release](https://github.com/ExchangeUnion/xud/releases), below an example using the `1.0.0-prealpha` release tag:
+```bash
+git clone https://github.com/ExchangeUnion/xud 
+cd xud
+git checkout tags/1.0.0-prealpha -b 1.0.0-prealpha
+npm install
+```
+
+
+`xud` uses [MySQL](https://www.mysql.com/) or [MariaDB](https://mariadb.org/). You will have to install one of those and create a user *xud* and a database *xud* and grant all permissions for the new database to the *xud* user.
+
+## 2. Prepare your environment 
+
+To initialize the database with some testing data, run the following command from `~/xud`:
+
+```bash
+npm run db:init
```

Comment:
In #252 it was discussed that this command is just for the test database although @itsbalamurali changed it in #244 I am confused now.

Reply:
Changes from #244 will be reverted, I adjusted the readme to make it clear.